ansible/playbooks/roles/compatibility_layer/tasks/install_prefix.yml

```
@@ -5,13 +5,13 @@
   fail:
     msg: |
       Error: the operating system of the installation host is {{ ansible_os_family }} {{ ansible_distribution_version }}.
-      The task for installing Gentoo Prefix currently only supports Linux distributions based on RHEL 8.
-  when: not(ansible_os_family == "RedHat" and ansible_distribution_major_version is version("8", "=="))
+      The task for installing Gentoo Prefix currently only supports Linux distributions based on RHEL 7.x or 8.x.
+  when: not(ansible_os_family == "RedHat" and (ansible_distribution_major_version is version("7", "==") or ansible_distribution_major_version is version("8", "==")))
```

```suggestion
  when: >
    not(
      ansible_os_family == "RedHat" and 
      (ansible_distribution_major_version is version("7", "==") or ansible_distribution_major_version is version("8", "=="))
    )
```
This should work. The `>` will replace the newlines by spaces (there's also `|`, which will keep the newlines).